### ISSUE TITLE
A new site section should not be able to be created at the root

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/cgov-digital-platform#3062]
### Issue description
A new site section should never be allowed to be created at the root - as this breaks our cancer.gov homepage. 

We need to block this from happening.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  A new site section is created and the landing page was set, with everything checked to hide in navigation. 
2. The parent page (in the Relations drop down) was not selected. 
3. Because there was no URL path,  the site section was added to the root
4. This caused our main homepage (cancer.gov) to not render and show a page not found status.

### What's the expected result?
-We need to make sure that a new site section is not able to be created at the root, and that a relations is set for any new site section that is not the root.

### What's the actual result?
- breakage of cancer.gov

### Additional details / screenshot
- ![Screenshot]()

### Related Tickets
* Issue #9999
* Issue #9999

**Comment by bryanpizzillo** [comment]
Added the constraint `You SHOULD never be allowed to have a term with 2 parents` to the requirements above. Currently, when saving, you could have multiple parents, one of which being Term 0, or indicating the root.

**Comment by sarinapadilla** [comment]
@alinai @VictoriaSunNIH @suryatheja-muppalla I've added the AC document to the ticket description above!

**Comment by lburack** [comment]
I updated the error message in the ACs per comments on the PR (https://github.com/NCIOCPL/cgov-digital-platform/pull/4304#issuecomment-2091163772)

Also this scenario should **NOT** be a thing. We have MANY, MANY site sections that have all navigation display options checked. This would break a ton of content on the enterprise and microsites. This needs to be removed from ACs and not a functional thing. Site sections are complicated and it's imperative that ACs be reviewed with me (or another lead IA) before development. 

**THIS IS BAD! THIS SCENARIO SHOULD NEVER HAVE BEEN CREATED. REMOVE FROM ACS!**
```gherkin
Scenario: User tries to save a new site section with invalid navigation display options
  Given the user has Site Admin permissions
  When a new site section is created 
    And the landing page was set 
    And all three check boxes in “Navigation Display Options” field is checked to hide in navigation 
  When the user clicks 'Save' on the site section
  Then the following error message appears: "The site section cannot be saved. A minimum of one (1) display option must be visible."
```

@andyvanavery31 @sarinapadilla @bryanpizzillo @navsunka @santwuma

**Comment by andyvanavery31** [comment]
@lburack I removed the AC's that you mentioned in your last comment. To confirm, we are allowing a site section to be completely hidden, and this occurs regularly across the enterprise and microsites? If yes, then we should be good with that one removed. Could you please let me know when you are able to review all of the AC's, and then @navsunka can continue development?

**Comment by lburack** [comment]
> @lburack I removed the AC's that you mentioned in your last comment. To confirm, we are allowing a site section to be completely hidden, and this occurs regularly across the enterprise and microsites? If yes, then we should be good with that one removed. Could you please let me know when you are able to review all of the AC's, and then @navsunka can continue development?

Yes @andyvanavery31 - this is correct. Thank you for removing that specific AC. The others look good to me. The only one I have a question about is the scenario where a user selects two parents – I don't think that's even possible to do right now. It's a single select option. But I'm on board with the outcome
